opentracing/src/main/java/opentracing/TraceCodec.java

```diff
@@ -24,16 +24,16 @@
  * binary message format, so that it can be inspected efficiently by the middleware / routing layers
  * without parsing the whole message.
  */
-public interface TraceContextCodec {
+public interface TraceCodec {
```

Comment:
From https://github.com/opentracing/opentracing-java/pull/7#issuecomment-173808490
> - serializes a Span to and from the wire (c.f. your enthusiasm for removing the Codec),

I'd envisioned that this type of information is specified by the Tracing platform, not by the OpenTracing specification.